dasharo/releases.yaml

```diff
@@ -1,24 +1,196 @@
 releases:
-        - version: v1.0.0
+        - version: v0.1.2
```

Comment:
Newer versions should be always on top. Please reverse the order.

Reply:
Fixed